### PR TITLE
`default-branch-button` - Restore feature in sidebar

### PR DESCRIPTION
**Pull request by kovsu**
close: #7549 

## Screenshot

When `default-branch-button` appears in the header, we can find the closet container element so it works fine!

image

But whetn it appears in the sidebar, the closet container element is null. And I think we don't need to add `rgh-z-index-5` class in this situation.

image

### CODE REVIEW

**Comment by kovsu** [comment]
after:

image

File: source/github-helpers/index.ts [hunk 1]
```diff
@@ -160,7 +160,11 @@ export function triggerConversationUpdate(): void {
 
 // Fix z-index issue https://github.com/refined-github/refined-github/pull/7430
 export function fixFileHeaderOverlap(child: Element): void {
-	child.closest('.container')!.classList.add('rgh-z-index-5');
```
Comment (fregante) [inline source/github-helpers/index.ts:163]: You can just turn the exclamation mark into ?.

Comment (kovsu) [reply source/github-helpers/index.ts:163]: Oh, I forget its ts file. hhhhh

File: source/github-helpers/index.ts [hunk 2]
```diff
@@ -160,7 +160,7 @@ export function triggerConversationUpdate(): void {
...
+	child.closest('.container')?.classList.add('rgh-z-index-5');
```
Comment (fregante) [inline source/github-helpers/index.ts:163]: We still want the comment here though, to clarify where it happens

**Review by fregante — APPROVED** [review]

File: source/github-helpers/index.ts [hunk 3]
```diff
@@ -160,7 +160,8 @@ export function triggerConversationUpdate(): void {
 
 // Fix z-index issue https://github.com/refined-github/refined-github/pull/7430
 export function fixFileHeaderOverlap(child: Element): void {
-	child.closest('.container')!.classList.add('rgh-z-index-5');
+	// When the default branch button appears in the sidebar, the closet container is null
```
Comment (fregante) [inline source/github-helpers/index.ts:163]:
```suggestion
	// In the sidebar the container is not present and this fix is not needed
```

**Pull request merged**